packages/styles/table.css

```diff
@@ -102,6 +103,10 @@
   box-shadow: var(--table-box-shadow);
 }
 
+.TableCell {
+  color: var(--gray-90);
+}
```

Comment:
This needs to only apply in light-mode. There are tons of color contrast issues otherwise.

<img width="1074" alt="image" src="https://user-images.githubusercontent.com/571265/214670416-259aaae6-537d-4cb9-bcc6-8585785c7353.png">